### ISSUE TITLE
Resize images

### ISSUE THREAD

**Issue by SR-G** [Bernardo-MG/docs-maven-skin#159]
Most of my images are shown in a way too wide size.

Of course i could resize them, but it would really be better to be able to resize them dynamically through Markdown, either image by image, either to have all images shown at max at 100% of column size.

Is there an easy way to something like this ?

**Comment by Bernardo-MG** [comment]
I'll take a look to it.

**Comment by Bernardo-MG** [comment]
Been making some experiments with https://github.com/Bernardo-MG/docs-bootstrap-template

Currently this would work correctly:

```html
<div class="row">
    <div>
         <a href="./static/img/Fronalpstock_big.jpg"><img class="img-responsive" src="static/img/Fronalpstock_big.jpg" alt="Fronalpstock"/></a>
    </div>
</div>
```

But the skin would build something like this, which doesn't work:


```html
<div class="row">
    <div>
        <figure>
            <a href="./static/img/Fronalpstock_big.jpg"><img class="img-responsive" src="static/img/Fronalpstock_big.jpg" alt="Fronalpstock"/></a>
            <figcaption>A big image</figcaption>
        </figure>
    </div>
</div>
```

Once I figure how to make a responsive image work inside a figure I'll update the skin.

**Comment by SR-G** [comment]
Nice findings. Good luck for finalizing.

Also i tryed to used `<img />` tags directly (instead of the markdown way to add images), but weirdly it seems to not been correctly rendered in the end (and the img tag is also included in a figure tag in that situation).

**Comment by Bernardo-MG** [comment]
The skin transforms the generated HTML to apply a bunch of fixes. It's not a neat solution, but the best I could manage with Maven Skin. For this reason any attemt to add HTML tags may fail.

I think I've managed to find a fix for the responsive images. the reason the figure tag was breaking when I tested it was due to this rule in the CSS files:

```css
figure {
    display: table;
    width: 1px;
    padding: 0.5em;
    margin-bottom: 1em;
    text-align: center;
    background-color: white;
    -webkit-box-shadow: 0 0 5px #cccccc;
    -moz-box-shadow: 0 0 5px #cccccc;
    box-shadow: 0 0 5px #cccccc;
}
```

Removing the "width: 1px;" line fixes it. I have no idea why I added that.

I'll fix it, verify it works and release a new version as soon as possible.

**Comment by Bernardo-MG** [comment]
I've just published version 1.4.2. Can you try it and tell me if it works?

**Comment by SR-G** [comment]
Hello. I've just tested it on my side and i can tell that it's now working fine (and that it's visually way better like this).
For example that wide mindmap was previously incredibly large - now it is reduced at column-width : https://i.imgur.com/OWVy62b.png

Thanks for the correction.
